### PR TITLE
Make sure revisioned table owner can add/drop columns

### PR DESCRIPTION
**Pull request by strk**
This PR is aimed at fixing bug #113

It starts with a failing test ...

### CODE REVIEW

**Comment by strk** [comment]
@palmerj reviews are not mandatory yet but I thought I could wait for yours here...
This change is needed to play nicely with the change introduced in #101

File: sql/10-table_add_column.sql [hunk 1]
```diff
@@ -11,23 +11,49 @@ DECLARE
...
     v_trigger_name TEXT;
     v_revision_table TEXT;
+    v_schema          NAME;
+    v_table           NAME;
+    v_owner           NAME;
 BEGIN
-    IF NOT @extschema@.ver_is_table_versioned(p_schema_name, p_table_name) THEN
-        RAISE EXCEPTION 'Table %.% is not versioned', quote_ident(p_schema_name), quote_ident(p_table_name);
+
+    SELECT
```
[the remark this answers is not in the document]
Comment (strk) [reply sql/10-table_add_column.sql]: a function to fetch schema name, table name and owner name from a table oid,  you mean ? May be a good idea, yes.

[the remark this answers is not in the document]
Comment (palmerj) [reply sql/10-table_add_column.sql]: > a function to fetch schema name, table name and owner name from a table oid, you mean ?

Could even involve the pg_has_role part, if the custom error message is passed as a parameter.

[the remark this answers is not in the document]
Comment (palmerj) [reply sql/10-table_add_column.sql]: > Right now the only common functions are in 14-common.sql which is loaded after 01-enable_versioniong.sql so should be somewhere in 001-comon.sql ...

Ah that's a problem having common functions run last! 

> to be honest I'm not sure if it is really worth to incapsulate such a small query into a function

Up to you if you want to action this - only a suggestion.

**Comment by strk** [comment]
@palmerj it takes formal approval of this to merge. For now I wouldn't generalize more.

**Pull request merged**